### DISCUSSION TITLE
Restarting a simulation

### DISCUSSION THREAD

**Discussion by remi-rc** [ProjectPhysX/FluidX3D#145] [Ideas]
Hi everyone,

I was wondering if there was currently a possibility to restart a simulation after it has stopped. My current case exceeds the time limit allowed on my cluster, so ideally I would then re-start from where I stopped !

I would assume that using the vtk exports, it could be feasible ?

Best,
Rémi

**Comment by ProjectPhysX** [comment]
Hi @remi-rc,

saving/loading checkpoints is not implemented, since I haven't yet had a case where runtime was limited, and runtime usually doesn't exceed a day. Checkpoints require dumping the entire LBM fields from VRAM to the hard drive, that is `rho`, `u`, `flags`, `f`, and extension fields. `.vtk` export is not sufficient, as you also need the density distribution functions`f`.

For now you could reduce resolution, which will significantly reduce runtime, or request a longer runtime limit from your admin if possible.

Kind regards,
Moritz

**Comment by remi-rc** [comment]
Hi @ProjectPhysX , 
thanks a lot for your answer !
Regards,
Rémi
